debian/rules.in

```diff
@@ -16,7 +16,7 @@ CUDA_OPT = $(shell echo ${DEB_BUILD_PROFILES} | grep -qw cuda \
 	dh $@ 
 
 override_dh_auto_configure:
-	./contrib/configure-release-mt --prefix=/usr \
+	../contrib/configure-release-mt --prefix=/usr \
```

Comment:
some tests fail because of this change

Reply:
@brminich Debian builds [fail ](https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=57005&view=logs&j=698c344f-0db0-55eb-b02a-d42dfa778b4b&t=581722d4-d92c-5e13-274b-3ed82c756e80&l=41)without this change, so maybe we should fix these tests.

Reply:
isn't [this](https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=57028&view=logs&j=a1437434-5193-5f40-7f41-b69248d83fa2&t=e254882d-22eb-5826-ce29-e3e5473c5857&l=37270) related?

Reply:
Resolved by the `sed` hack for now.